exotica/src/Scene.cpp

```diff
@@ -504,7 +504,7 @@ void Scene::loadSceneFile(const std::string& file_name, const Eigen::Isometry3d&
 
 void Scene::loadSceneFromStringStream(std::istream& in, const Eigen::Isometry3d& offset, bool updateCollisionScene)
 {
-    ps_->loadGeometryFromStream(in, Eigen::Affine3d(offset));
+    ps_->loadGeometryFromStream(in, Eigen::Affine3d(offset.affine()));
```

Comment:
Dont do this. I am about to make a proper fix. The reason is MoveIt diverged the API last night between Melodic (upcoming, not yet released) and Indigo, Kinetic, and currently released Melodic.